tools/bin/ariaDownload.py

```diff
@@ -17,7 +17,10 @@
 import requests
 import argparse
 from datetime import datetime
+import logging
+from ARIAtools.logger import logger
```

Comment:
Codacy found an issue: ['ARIAtools.logger.logger' imported but unused (F401)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5957665)

Reply:
@Askaholic i understand from @bbuzz31 that this is correct and essential for the logging?